src/acquisition/covidcast/csv_importer.py

```diff
@@ -271,13 +279,62 @@ def extract_and_check_row(row, geo_type):
 
     # optional not-too-small float
     try:
+      # Returns either a float, None, or causes an exception
       sample_size = CsvImporter.maybe_apply(float, row.sample_size)
     except ValueError:
       # expected a number, but got a string
       return (None, 'sample_size')
 
+    if hasattr(row, "missing_val"):
```

Comment:
is hasattr needed if the row is initialized with these values?

Reply:
This is backwards compatibility thing for CSVs that don't have the `missing` columns. `row` should just be an unprocessed line from a recent `read_csv` call, so it will have whatever the CSV has.

Reply:
I suppose there's a question of whether to strictly enforce the presence of missing columns or to impute them if they're missing. I think I'm leaning towards making them required now... this way the NAN coding is contained in one place (mostly), instead of two different codebases

Reply:
got it, makes sense.